server/http.go

```diff
@@ -915,3 +984,45 @@ func (p *Plugin) slackAttachmentToUpdatePMI(currentValue, channelID string) *mod
 
 	return &slackAttachment
 }
+
+func (p *Plugin) checkChannelPreference(channelID string) (bool, int, error) {
```

Comment:
```suggestion
func (p *Plugin) isChannelRestrictedForMeetings(channelID string) (bool, int, error) {
```

Not sure why we need the http status code to be returned from here?

Reply:
Returning `statusCode` for any error that arises so we can return the respective error.